sagemaker_model_monitor/fairness_and_explainability_jsonlines/SageMaker-Monitoring-Bias-Drift-for-Endpoint.ipynb

```diff
@@ -0,0 +1,1949 @@
+{
```

Comment:
<p>Do we not want to add contents in these notebooks? I feel they are useful when preferred choice of platform is Jupyter instance.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3760/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>Indeed, I will add the table of content which is also useful for reading the notebook on readthedocs website. </p>